### ISSUE TITLE
Error in `UpdateSeuratForSemla()`: ! Only 'SlideSeq' and 'VisiumV1' are currently supported. Got VisiumV2 Run `rlang::last_trace()` to see where the error occurred.

### ISSUE THREAD

**Issue by wuxiangning** [ludvigla/semla#41]
Hello, thank you very much for providing such a convenient tool. 
However, I encountered an error while converting a 2021 dataset to a selma file, and I am not sure what the reason is. Below is the information about my Seurat object. Thank you very much!
![{214BAF1E-ED7B-4D96-8529-425B8AB5A4B2}](https://github.com/user-attachments/assets/5006e8cc-87a5-4ebc-9bd8-15de0418a78e)
![{2C5EE462-68F9-45E0-84D8-B182AE037A58}](https://github.com/user-attachments/assets/6a892d36-608e-4ab3-b8ce-2281aa46f681)

**Comment by wuxiangning** [comment]
![1729825108046](https://github.com/user-attachments/assets/65b95fcc-0083-4445-8c6b-96573a9ae8ba)
I only encountered this issue when running the 2021 dataset, while the conversion works normally with other datasets. I’m not sure why this is happening and would appreciate your guidance. Thank you!

**Comment by jemorlanes** [comment]
Hello!

Thanks for using the package :). We are currently looking at this issue (see Issue [40)](https://github.com/ludvigla/semla/issues/40) and will make a follow up comment once the changes are implemented.

**Comment by jemorlanes** [comment]
Hello! We have not merged into the main branch the updated version of `UpdateSeuratForSemla`, but we have a branch that has solved that issue.

Reinstall semla using `remotes::install_github("ludvigla/semla", ref = "upd_seuTosem")` and let us know if you it runs successfully! In the coming days we will merge this branch into `main`, but for now this should work. We will comment again once it is merged.

**Comment by wuxiangning** [comment]
Hello!It works now!
Thank u!
![image](https://github.com/user-attachments/assets/e38c9c60-81e1-4fb0-83e0-fa7fa2d204a6)

**Comment by jemorlanes** [comment]
Hi! We have now merged the branch into main :))
